providers/gce/OWNERS

```diff
@@ -1,6 +1,5 @@
 # See the OWNERS docs at https://go.k8s.io/owners
-
-approvers:
+emeritus_approvers:
```

Comment:
You seem to have removed all approvers for this directory. Do we want that?

Reply:
Thanks for catching!(It's been removed from legacy-cloud-providers/gce.) I have added back for //providers/gce/OWNERS.